doc/dev/documenting.rst

```diff
@@ -2,6 +2,20 @@
  Documenting Ceph
 ==================
 
+User documentation
+==================
+
+The documentation on docs.ceph.com is generated from the restructuredText
+sources in ``/doc/`` in the Ceph git repository.
```

Comment:
Speaking of this, is there an easy way to see a preview of the rendered doc? If so, we should include that here.